includes/class-amp-story-post-type.php

```diff
@@ -166,6 +166,9 @@ public static function register() {
 		// Register the Latest Stories block.
 		add_action( 'wp_loaded', array( __CLASS__, 'register_block_latest_stories' ), 11 );
 
+		// The AJAX handler for when an image is cropped and sent via POST.
+		add_action( 'wp_ajax_custom-header-crop', array( __CLASS__, 'crop_featured_image' ) );
```

Comment:
Can we use a different Ajax action name here? This would help distinguish it from the one used by `Custom_Image_Header`.

Reply:
Yeah, it's strange having this name that references the custom image header.

But it looks like the [wp.ajax.post call](https://github.com/WordPress/wordpress-develop/blob/6b366c6620fdd5960cedcdf80955966a715efa82/src/js/media/controllers/cropper.js#L158) that uses the `'custom-header-crop'` action is in another file, so there's not an easy way to change that.